### PR TITLE
Design

### PR DESCRIPTION
**Pull request by domicode**

### CODE REVIEW

File: Gemfile.orig [hunk 1]
```diff
@@ -0,0 +1,47 @@
+source 'https://rubygems.org'
+ruby '2.2.2'
+
+# essentials
+gem 'rails', '4.2.1'
+gem 'multi_json'
+gem 'pg'
+
+# frontend
+<<<<<<< Updated upstream
```
Comment (masone) [inline Gemfile.orig]: unresolved merge conflict

**Comment by masone** [comment]
This pull request contains changes from pull request #14. I have reviewed the changes there, it needs some work. Please create a pull request with your isolated changes and don't mix yours up with changes from other prs.

**Comment by domicode** [comment]
sorry, this PR should have been sent to the pantertech fork. Somehow I reopened it by accident as I choose the wrong fork.

**Comment by masone** [comment]
as this is the main repo, you should submit your prs here. not on a fork.

**Pull request closed without merging**